src/Connection.php

```diff
@@ -404,11 +406,12 @@ public function rollBack()
      *
      * Few Connection drivers need to receive Model to get ID because PDO doesn't support this method.
      *
-     * @param \atk4\data\Model Optional data model from which to return last ID
+     * @param Model Optional data model from which to return last ID
+     * @param string Optional sequence name from which to return last ID (this takes precedence)
      *
      * @return mixed
      */
-    public function lastInsertID($m = null)
+    public function lastInsertID(Model $m = null, string $sequence = null)
```

Comment:
We are at atk4/dsql here, 1st param should be completely removed.

Reply:
Hmm. In theory yes - atk4/dsql should not rely on atk4/data.
But on other hand - see Connection_Oracle and Connection_PgSQL. How can I get lastInsertID from $model->sequence if dsql will know nothing about models ?

Reply:
https://github.com/atk4/dsql/pull/195/files#diff-ae503101b8b31c128532e45a73ea0dc0R28

this should be put in data/Model and passed here as a string, no other usage here

Reply:
OK, you're right. Will make a fix here and also small relates PR in data now.